packages/core/src/plugins/meta/audius.ts

```diff
@@ -128,10 +129,13 @@ class AudiusMetaProvider extends MetaProvider {
     throw new Error('Method not implemented.');
   }
 
-
   async fetchAlbumDetailsByName(): Promise<AlbumDetails> {
     throw new Error('Method not implemented.');
   }
+
+  searchForPodcast(): Promise<SearchResultsPodcast[]> {
```

Comment:
@nukeop is it possible to make an interface for `MetaProvider` to implement default method to return empty value. So we don't need to do like this in very meta provider when we come ups with new method.

Reply:
@Ivan12273 You can add this method to this abstract class with a default implementation: https://github.com/nukeop/nuclear/blob/6a99b3f85042f170de840d3e9b9381d8cd5fd84b/packages/core/src/plugins/metaProvider.ts

This way you can remove it from child providers.

Reply:
I added the method with the default implementation and removed it from the others classes. I needed to add` // eslint-disable-next-line @typescript-eslint/no-unused-vars` for the unused var :)